content/material_and_questions.md

```diff
@@ -44,4 +53,151 @@ In the below table, any minimum or maximum variation between material years are
+Context is defined as the verses or verses that are required.
+
+##### Quote
+A single verse
+The Question Type abbreviation is "QT"
+
+##### Quote-these-Two-Verses
+Two consecutive verses
+The Question Type abbreviation is "Q2V"
+Should be verses that make more sense when quoted together.  Example: Gal 5:22-23 
+
+#### Finish
+Will be direct quotes from Scripture
+Must be quoted word-perfect
+Must be strong enough to stand on its own without requiring additional verses to explain it.
+Must be spiritually significant.
+The first five words must be unique to all Finish questions.
+The quizmaster will read the first five words of the question (less if a quizzer jumps)
+Quizzers may go back and correct their answers if done in the 30-second time limit.  The quizzer will be required to make one full correct rotation. However, the quizzer can be called correct if they have made a mistake only in one area of the verse AND correct the mistake making it abundantly clear to the quiz master and answer judge that they know where their mistake was, and what he corrected words are.
+Context is defined as the verses or verses that are required.
+
+##### Finish-the-Verse
+Begins at the beginning of a single verse and finishes at the end of that verse.
+The Question Type abbreviation is "FTV"
+
+##### Finish-This
+Begins in the middle of a verse and finishes at the end of the verse.  A FT must start at the beginning of a sentence or the beginning of a quotation.
+The Question Type abbreviation is "FT"
+
+##### Finish-these-Two-Verses
+Begins at the beginning of a verse and continues to the end of the next verse. Used when a verse is not strong enough on its own and needs another verse to explain it or can be used when two verses combined form an even stronger thought. 
+The Question Type abbreviation is "F2V"
+
+##### Finish-This-and-the-Next
+Begins in the middle of a verse and finishes at the end of the next verse.  A FTN must start at the beginning of a sentence or the beginning of a quotation. Used when part of a verse is not strong enough on its own and needs another verse (or part of a verse) to explain it or can be used when two verses combined form an even stronger thought.
+The Question Type abbreviation is "FTN"
+
+FTV questions should not start out with an insignificant introduction such as “He replied” or “He said to them”.  These verses, if spiritually significant, should be used as FT or FTN type questions.  For example, Luke 11:28 says, He replied, “Blessed rather are those who hear the word of God and obey it.”  This verse could be used as an excellent FT question starting with “Blessed rather are those who …”. 
+
+#### Situation
```

Comment:
I'm very proud of this section. I think it's so much clearer.

Reply:
I like it, but it's not func equiv to the current rulebook. I'd save it off to some place for inclusion in v-next. And when it gets included, it should probably be included under one of the subheaders that get collapsed in default "simple view".

Reply:
What is not functionally equivalent?

Reply:
Is "FTV questions should not start out with an insignificant introduction" a concept defined in the current rulebook? For example, if the first 5 words of a verse start "He replied" but are globally unique, is there anything in the current rulebook that prevents a FTV starting with "He replied"?

Reply:
that whole section is verbatim from the current rule book.   But because under the QM section there is a distinction between "should" and "must", I would assume here that "should not" != "must not" which would make this bit more of a guideline/recommendation, than an objective rule that can be challenged if not followed.

Which is very problematic. 

But there are squirrelly use cases for sure...."insignificant introduction" is difficult to define objectively. "He said to them" is pretty insignificant.  "He said to a few of them" may NOT be "insignificant."

Reply:
I'd go with adding this to a best practices doc, even though I think in v-next it should be included as a must. I'd prefer to include it in v1 as a must, but then we break FE. And if we include best practices in v1, we're compromising on one of our v1 goals, I think.

Reply:
agree to ALL of that, unfortunately. It belongs in a Best Practices doc. If people agree on a way to make the wording objective and robust, then it belongs as a MUST in the rule book. 

But for now I think we gotta roll with it, as-is, in the rule book.

Reply:
Do we? I thought v1 is a rulebook at FE with best practices removed. (This would probably be an easier discussion over Zoom/Hangouts/Skype at some point when we can get the 4 of us together, since it's about project philosophy and v1 scope.)

Reply:
hmmmm, I didn't think BP were getting removed from v1. 

Happy to hop on a call now, though it'd be just the two of us.

Reply:
Removing BP means removing:
* all the Q/F "stand on its own, significant" stuff
* INT not being overly long
* FTV not starting with "insignificant introduction"
* "2V" being on verses that "make more sense when quoted together"

and probably a LOT more

Reply:
Yep. Agreed. I think that stuff needs to get moved to best practices. I'm happy to be overruled on this one. But I think if it can be moved to BP without impacting FE, then it should be. We'll publish a (rough and unpolished) BP in parallel with v1 of the rule book, so the content will still exist.

Reply:
Moving all BP to FE is what you, Zach, and myself agreed on in the meeting yesterday right?

Reply:
Yes, I think so.

Reply:
I'm concentrating on pure content and FE in these initial 5 PRs by me. 

I'll likely take my own OCD pass (unless you want all that fun to yourself Gryphon).

I will also take a "removing BP" pass, after these initial 5 are merged.

Reply:
I think you taking an OCD pass before me will be a very good thing. My OCD pass will be purely grammatical, syntactical, and stylistic.

Reply:
@gryphonshafer going back to the first two comments in this thread......I meant my initial comment to refer to the Situation question section. Is your reply regarding that? Or were you talking about the FTV/FT BP language?

Reply:
I meant the FTV/FT BP language.